sites/nostv.pt/nostv.pt.channels.xml

```diff
@@ -160,7 +160,7 @@
   <channel site="nostv.pt" lang="pt" xmltv_id="TVCineTop.pt" site_id="141">TVCine TOP HD</channel>
   <channel site="nostv.pt" lang="es" xmltv_id="TVEInternacionalEuropeAsia.es" site_id="48">TVEi</channel>
   <channel site="nostv.pt" lang="pt" xmltv_id="TVI.pt" site_id="555">TVI HD</channel>
-  <channel site="nostv.pt" lang="pt" xmltv_id="TVIFiccao.pt" site_id="574">TVI Ficção HD</channel>
+  <channel site="nostv.pt" lang="pt" xmltv_id="VmaisTVI.pt" site_id="574">V+ TVI</channel>

```

Comment:
Invalid xmltv_id.

https://github.com/iptv-org/epg/blob/master/CONTRIBUTING.md#how-to-add-a-channel-to-the-guide

Reply:
It literally isnt invalid. The channel changed name and programming. You should accept it.